card/mod/admin/spec/set/self/admin_info_spec.rb

```diff
@@ -0,0 +1,11 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::Set::Self::AdminInfo do
+  # SPECSTUB
```

Comment:
SPECSTUB means there aren't any specs yet.  So we can delete these.

Reply:
ok